webpack.config.js

```diff
@@ -29,7 +29,7 @@ const WEBPACK_MODE = IS_PRODUCTION ? 'production' : 'development';
 const SENTRY_BACKEND_PORT = env.SENTRY_BACKEND_PORT;
 const SENTRY_WEBPACK_PROXY_PORT = env.SENTRY_WEBPACK_PROXY_PORT;
 const USE_HOT_MODULE_RELOAD =
-  !IS_PRODUCTION && SENTRY_BACKEND_PORT && SENTRY_WEBPACK_PROXY_PORT;
+  !IS_PRODUCTION && SENTRY_BACKEND_PORT && SENTRY_WEBPACK_PROXY_PORT && !!env.HOT_RELOAD;
```

Comment:
This `!IS_PRODUCTION` part should probably be replaced with @priscilawebdev's new `IS_DEV_ENV`. Also `!!` is considered harmful per that same PR :) Finally, I'd namespace this new env variable: `SENTRY_FE_HOT_RELOAD` as we can have hot reload for backend too.

```suggestion
  !IS_PRODUCTION && SENTRY_BACKEND_PORT && SENTRY_WEBPACK_PROXY_PORT && Boolean(env.SENTRY_FE_HOT_RELOAD);
```

Reply:
So I checked on that again, and they are actually equivalent, @priscilawebdev's example compared `!var` with `Boolean(var)`, when the equivalent comparison would be `!!var === Boolean(var)`.

Reply:
I updated `Boolean()` --> `!!` because we use it more in the codebase (there's only one instance where we use `Boolean` to cast)

Reply:
>I updated Boolean() --> !! because we use it more in the codebase (there's only one instance where we use Boolean to cast)

https://eslint.org/docs/rules/no-implicit-coercion is one of my favorite rules but apparently we don't follow it so 👍 for consistency :)